extension.json

```diff
@@ -15,18 +16,50 @@
 	"requires": {
 		"MediaWiki": ">= 1.27"
 	},
+	"AutoloadClasses": {
+		"SFS\\Hooks": "SemanticFormsSelect.hooks.php",
+		"SFS\\SemanticFormsSelectInput": "src/SemanticFormsSelectInput.php",
+		"SFS\\Output": "src/Output.php",
+		"SFS\\SelectField": "src/SelectField.php",
+		"SFS\\ApiSemanticFormsSelectRequestProcessor": "src/ApiSemanticFormsSelectRequestProcessor.php",
+		"SFS\\ApiSemanticFormsSelect": "src/ApiSemanticFormsSelect.php"
+	},
+	"config": {
+		"ScriptSelectCount": 0,
+		"SF_Select_debug": false
+	},
+	"APIModules": {
+		"sformsselect": "SFS\\ApiSemanticFormsSelect"
```

Comment:
I am not a fan of having PHP class names in this JSON file. Referencing them in PHP with ::class is much more tool friendly.

Reply:
The only way I see to handle this is recovering SemanticFormsSelect.php file and doing it there. Let me know...

Reply:
My comment was mostly an educational FYI. I don't think this should be a blocker for this PR.

You can use the callback thing in extension.json to put some code in a static function of a class that does things like API module registration. By having it in a class it will get autoloaded and you don't need to include the file via the `file` section in `composer.json`, which causes some problems.

https://github.com/JeroenDeDauw/Maps/blob/master/extension.json#L18

The problem of putting PHP in JSON here is especially bad if there are function names involved, such as with the `SFS\\Hooks::onSemanticFormsSelectSetup` below.